code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -440,46 +440,3 @@
 		M.emote(pick("twitch","laugh","frown"))
 	..()
 	. = 1
-
-/datum/reagent/drug/pumpup
-	name = "Pump-Up"
-	description = "Take on the world! A fast acting, hard hitting drug that pushes the limit on what you can handle."
-	reagent_state = LIQUID
-	color = "#e38e44"
-	metabolization_rate = 2 * REAGENTS_METABOLISM
-	overdose_threshold = 30
```

Comment:
you might be better off messing with this number

Reply:
OD counters are good for ingrained chemicals that won’t be removed. Chemicals that have multiple uses, beneficial ones, but are seen frequently abused for combat use. Pump up exists to literally just be used a combat chem with no uses whatsoever ourside of that. Therefore I see no reason to not remove it. Plus even with OD stuff people can just manage chem amount and abuse it that way.